R/gitbook.R

```diff
@@ -1,7 +1,10 @@
 #' The GitBook output format
 #'
+#' @description
 #' This output format function ported a style provided by GitBook
-#' (\url{https://www.gitbook.com}) for R Markdown.
+#' (\url{https://www.gitbook.com}) for R Markdown. To read more about this format, see:
+#' https://bookdown.org/yihui/bookdown/html.html#gitbook-style
```

Comment:
If we want this to be a clickable url it needs to be written at such. 

With current style it would be
```suggestion
#' \url{https://bookdown.org/yihui/bookdown/html.html#gitbook-style}
```